<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CGexFtpSettings_widget_base</class>
 <widget class="QWidget" name="CGexFtpSettings_widget_base">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>619</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FTP settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="HBoxLayout_1">
     <item>
      <widget class="QLabel" name="labelProfileNameCombo">
       <property name="text">
        <string>Profile name:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboProfileName">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Select profile to display or 'New settings...' to create a new profile</string>
       </property>
       <property name="duplicatesEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonPrevProfile">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Previous profile</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="gex_ftp.qrc">
         <normaloff>:/gex-ftp/images/resources/arrow_left.png</normaloff>:/gex-ftp/images/resources/arrow_left.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonNextProfile">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Next profile</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="gex_ftp.qrc">
         <normaloff>:/gex-ftp/images/resources/arrow_right.png</normaloff>:/gex-ftp/images/resources/arrow_right.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer41">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>81</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxFtpSettings">
     <property name="title">
      <string>Ftp settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="_14">
        <item>
         <widget class="QLabel" name="labelProfileName">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>32767</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;font color=&quot;#5555ff&quot;&gt;&lt;b&gt;&lt;i&gt;Profile Name:&lt;/i&gt;&lt;/b&gt;&lt;/font&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editSettingsName">
          <property name="toolTip">
           <string>Enter profile name</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="Line" name="line1">
        <property name="frameShape">
         <enum>QFrame::HLine</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="checkBoxProfileEnabled">
        <property name="text">
         <string>Profile enabled</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="labelURL">
          <property name="text">
           <string>Ftp site URL:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>comboURL</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="_13">
          <item>
           <widget class="QComboBox" name="comboURL">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
            <property name="toolTip">
             <string>Enter FTP server URL (use IP address if known)</string>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
            <property name="autoCompletion">
             <bool>true</bool>
            </property>
            <property name="duplicatesEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelPort">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Port:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinPortNb">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="toolTip">
             <string>Enter port to use on FTP server (usually 21)</string>
            </property>
            <property name="maximum">
             <number>999</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelLogin">
          <property name="text">
           <string>Login:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>editLogin</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="editLogin">
          <property name="toolTip">
           <string>Enter user login to use on FTP server</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelPassword">
          <property name="text">
           <string>Password:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>editPassword</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="editPassword">
          <property name="toolTip">
           <string>Enter password to use on FTP server</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelRemoteDir">
          <property name="text">
           <string>Remote folder:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>editRemoteDir</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="editRemoteDir">
            <property name="toolTip">
             <string>Select remote directory on FTP server (&quot;/&quot; is the root folder)</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonBrowseRemoteDirectory">
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>26</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::TabFocus</enum>
            </property>
            <property name="toolTip">
             <string>Browse</string>
            </property>
            <property name="whatsThis">
             <string>Browse to select local directory</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonTestRemoteFolder">
            <property name="text">
             <string>Test Folder</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QCheckBox" name="checkBoxRecurse">
            <property name="toolTip">
             <string>Check this option if you want Gex-Ftp to recursively scan all subfolders, and download all matching files.</string>
            </property>
            <property name="text">
             <string>Scan sub-folders</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="labelLocalDir">
          <property name="text">
           <string>Local folder:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>editLocalDir</cstring>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <layout class="QHBoxLayout" name="_11">
          <item>
           <widget class="QLineEdit" name="editLocalDir">
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="toolTip">
             <string>Enter local folder to store retrieved files</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonBrowseLocalDirectory">
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>26</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::TabFocus</enum>
            </property>
            <property name="toolTip">
             <string>Browse</string>
            </property>
            <property name="whatsThis">
             <string>Browse to select local directory</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="labelFileExtensions">
          <property name="text">
           <string>File extensions:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>editFileExtensions</cstring>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="editFileExtensions">
          <property name="toolTip">
           <string>Enter extensions of files to transfer</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="labelFilePolicy">
          <property name="text">
           <string>File policy:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>comboFilePolicy</cstring>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <layout class="QHBoxLayout" name="_12">
          <item>
           <widget class="QComboBox" name="comboFilePolicy">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="focusPolicy">
             <enum>Qt::WheelFocus</enum>
            </property>
            <property name="toolTip">
             <string>What should be done with files on remote server once they have been transferred to the local system</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelPostFilePolicy">
            <property name="text">
             <string>after downloaded from server</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="8" column="1">
         <widget class="QCheckBox" name="checkBoxDisabledDateTimeCheck">
          <property name="text">
           <string>Disable check on modification date/time when transferring files</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="labelSorting">
            <property name="text">
             <string>Download files in following order:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboDownloadOrder"/>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBoxDateWindow">
        <property name="toolTip">
         <string>Only transfer files with modification date/time in specified date window</string>
        </property>
        <property name="title">
         <string>Date window</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QGridLayout" name="_15">
           <item row="0" column="2">
            <widget class="QPushButton" name="buttonFromCalendar">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>24</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>24</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="gex_ftp.qrc">
               <normaloff>:/gex-ftp/images/resources/calendar_icon.png</normaloff>:/gex-ftp/images/resources/calendar_icon.png</iconset>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="labelFrom">
             <property name="text">
              <string>From:</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLabel" name="labelToDateTime">
             <property name="text">
              <string>&lt;To date/time&gt;</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labelTo">
             <property name="text">
              <string>To:</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="buttonToCalendar">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>24</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>24</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="gex_ftp.qrc">
               <normaloff>:/gex-ftp/images/resources/calendar_icon.png</normaloff>:/gex-ftp/images/resources/calendar_icon.png</iconset>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="labelFromDateTime">
             <property name="text">
              <string>&lt;From date/time&gt;</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QDateTimeEdit" name="dateTimeEditFrom">
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QDateTimeEdit" name="dateTimeEditTo">
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <spacer name="spacer6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>98</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>38</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="0" colspan="2">
       <layout class="QHBoxLayout" name="_9">
        <item>
         <widget class="QPushButton" name="buttonApply">
          <property name="focusPolicy">
           <enum>Qt::TabFocus</enum>
          </property>
          <property name="toolTip">
           <string>Apply settings to current profile</string>
          </property>
          <property name="text">
           <string>Apply</string>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonDelete">
          <property name="focusPolicy">
           <enum>Qt::TabFocus</enum>
          </property>
          <property name="toolTip">
           <string>Delete current profile</string>
          </property>
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer20">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>205</width>
            <height>16</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="buttonHelp">
          <property name="focusPolicy">
           <enum>Qt::TabFocus</enum>
          </property>
          <property name="toolTip">
           <string>Display help on Ftp settings</string>
          </property>
          <property name="text">
           <string>Help</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>comboProfileName</tabstop>
  <tabstop>editSettingsName</tabstop>
  <tabstop>comboURL</tabstop>
  <tabstop>editLogin</tabstop>
  <tabstop>editPassword</tabstop>
  <tabstop>editRemoteDir</tabstop>
  <tabstop>buttonBrowseLocalDirectory</tabstop>
  <tabstop>editFileExtensions</tabstop>
  <tabstop>buttonApply</tabstop>
  <tabstop>buttonDelete</tabstop>
  <tabstop>buttonHelp</tabstop>
  <tabstop>buttonPrevProfile</tabstop>
  <tabstop>buttonNextProfile</tabstop>
  <tabstop>checkBoxProfileEnabled</tabstop>
  <tabstop>groupBoxDateWindow</tabstop>
  <tabstop>dateTimeEditFrom</tabstop>
  <tabstop>buttonFromCalendar</tabstop>
  <tabstop>dateTimeEditTo</tabstop>
  <tabstop>buttonToCalendar</tabstop>
 </tabstops>
 <resources>
  <include location="gex_ftp.qrc"/>
 </resources>
 <connections/>
</ui>
